tests/par-test.cc

```diff
@@ -0,0 +1,59 @@
+/**
+ * CMA-ES, Covariance Matrix Adaptation Evolution Strategy
+ * Copyright (c) 2014 Inria
+ * Author: Emmanuel Benazera <emmanuel.benazera@lri.fr>
+ *
+ * This file is part of libcmaes.
+ *
+ * libcmaes is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * libcmaes is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with libcmaes.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "cmaes.h"
+#include <iostream>
+#include <chrono>
+#include <thread>
+
+#ifdef HAVE_TBB
+// Quick hack for definition of 'I' in <complex.h>
+#undef I
+#include <tbb/task_scheduler_init.h>
+#endif
+
+using namespace libcmaes;
+
+FitFunc fsphere = [](const double *x, const int N)
+{
+  double val = 0.0;
+  for (int i=0;i<N;i++)
+    val += x[i]*x[i];
+  std::this_thread::sleep_for(std::chrono::milliseconds(10));
+  return val;
+};
+
+int main(int argc, char *argv[])
+{
+#ifdef HAVE_TBB
+  static tbb::task_scheduler_init init;
+#endif
+  int dim = 10; // problem dimensions.
+  std::vector<double> x0(dim,10.0);
+  double sigma = 0.1;
+  //int lambda = 100; // offsprings at each generation.
+  CMAParameters<> cmaparams(x0,sigma);
+  cmaparams.set_mt_feval(true);
+  //cmaparams._algo = BIPOP_CMAES;
+  CMASolutions cmasols = cmaes<>(fsphere,cmaparams);
+  std::cout << "best solution: " << cmasols << std::endl;
+  std::cout << "optimization took " << cmasols.elapsed_time() / 1000.0 << " seconds\n";
+}
```

Comment:
return 0 ?

Reply:
> return 0 ?

All other tests have the same structure. Not returning anything; I kept it for this one too.

Reply:
Yes but's that incorrect and the compiler may throw warnings.

Reply:
> Yes but's that incorrect and the compiler may throw warnings.

Fixed